README.rst

```diff
@@ -31,6 +31,8 @@ development machine:
 
 - `Docker <https://www.docker.com/>`__
 
+Note: CMake is required for XGBoost. If using macOS, install CMake (pip install cmake)
```

Comment:
Would it be better for Mac users to install CMake via brew? `brew install cmake`

Reply:
pip comes default with macOS so chose pip over brew